src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -17,7 +17,7 @@
     get_one_of_subscription_locations,
     get_resource_name_completion_list)
 from azure.mgmt.compute.models import ContainerServiceOchestratorTypes
-from azure.cli.command_modules.vm._validators import _validator_ssh_key
+from azure.cli.command_modules.vm._validators import _validate_ssh_key
```

Comment:
The rest of the validators don't start with underscore, so you could drop the leading _.